lib/php/rendition-factory/src/Transformer/Document/DocumentToPdfTransformerModule.php

```diff
@@ -17,6 +19,28 @@ public static function getName(): string
         return 'document_to_pdf';
     }
 
+    public static function getDocumentation(): Documentation
+    {
+        static $doc = null;
+        if (null === $doc) {
```

Comment:
It's not necessary to cache it, it should be called only once by the documentation generator.
Also it's not very consuming so let's free memory!